src/SQLite.Net/SQLiteConnection.cs

```diff
@@ -335,12 +335,8 @@ public int CreateTable(Type ty, CreateFlags createFlags = CreateFlags.None)
             sbQuery.Append(")");
             var count = Execute(sbQuery.ToString());
 
-            if (count == 0)
-            {
-                //Possible bug: This always seems to return 0?
-                // Table already exists, migrate it
-                MigrateTable(map);
-            }
+            // add columns if necessary
+            MigrateTable(map);
```

Comment:
does this safely no-op?

Reply:
Yes, it's an empty for loop.  In fact, if you read the old comment, this was known to be happening when they didn't want it to.